client/executor/common/src/runtime_blob/runtime_blob.rs

```diff
@@ -81,6 +81,19 @@ impl RuntimeBlob {
 		export_mutable_globals(&mut self.raw_module, "exported_internal_global");
 	}
 
+	/// Perform an instrumentation that makes sure that a specific function `entry_point` is exported
+	pub fn entry_point_exists(&self, entry_point: &str) -> bool {
+		if let Some(v) = self.raw_module.export_section().map(|e| {
+			e.entries()
+			.iter()
+			.any(|e| matches!(e.internal(), Internal::Function(_)) && e.field() == entry_point)
+		}) {
+			v
+		} else {
+			false
+		}
```

Comment:
```suggestion
		self.raw_module.export_section().map(|e| {
			e.entries()
			.iter()
			.any(|e| matches!(e.internal(), Internal::Function(_)) && e.field() == entry_point)
		}).unwrap_or_default()
```

Why didn't you applied my previous suggestion?

Reply:
It wouldn't build on my computer. The dual .iter().iter() was not working for me. Possibly an error on my part.